Tools/peg_generator/Makefile

```diff
@@ -41,6 +43,12 @@ regen-metaparser: pegen/metagrammar.gram pegen/*.py
 
 .PHONY: test
 
+venv:
+	$(PYTHON) -m venv $(VENVDIR)
+	$(VENVPYTHON) -m pip install -U pip setuptools
+	$(VENVPYTHON) -m pip install -U memory_profiler
```

Comment:
Shouldn't this have been `-r requirements.pip`? (And isn't setuptools automatically installed by venv?)

Reply:
> And isn't setuptools automatically installed by venv?

Yes, but not updated to the latest version

> Shouldn't this have been `-r requirements.pip`?

Yep, I will do a PR to fix it

Reply:
Opened https://github.com/python/cpython/pull/20194